src/main/java/dev/j3fftw/litexpansion/items/CargoConfigurator.java

```diff
@@ -94,6 +103,16 @@ public void onCargoConfiguratorItemClick(PlayerInteractEvent e) {
             return;
         }
 
+        if (SlimefunItem.getByItem(Items.CARGO_CONFIGURATOR).isDisabled()) {
```

Comment:
We already do an item lookup before with `isItem`. Condense these into one (especially since it shouldn't run anything if disabled) save an item lookup